packages/terra-form-select/src/_FrameUtil.js

```diff
@@ -15,24 +15,24 @@ class FrameUtil {
   /**
    * Determines the dropdown position.
    * @param {Object} props - The component props.
-   * @param {Object} state - The component state.
    * @param {ReactNode} target - The select wrapper.
    * @param {ReactNode} dropdown - The dropdown.
+   * @param {number} maxHeight - The maxHeight of the dropdown dropdown.
    * @return {Object} - The calculated dropdown attributes.
    */
-  static dropdownPosition(props, state, target, dropdown) {
+  static dropdownPosition(props, target, dropdown, maxHeight) {
     const style = Object.assign({}, props).style || {};
     const { height } = dropdown.getBoundingClientRect();
     const { bottom, width, top } = target.getBoundingClientRect();
 
     const spaceBelow = window.innerHeight - bottom;
-    const maxHeight = style.maxHeight ? parseInt(style.maxHeight, 10) : Infinity;
-    const canFitBelow = maxHeight < spaceBelow || height < spaceBelow || spaceBelow > top;
+    const maximumHeight = parseInt(style.maxHeight || maxHeight, 10) || Infinity;
+    const canFitBelow = maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top;
     const availableSpace = canFitBelow ? spaceBelow : top;
 
     return {
       width,
-      maxHeight: Math.min(maxHeight, availableSpace - 10),
+      maxHeight: Math.min(maximumHeight, availableSpace - 10),
```

Comment:
Not really related to these changes, but what is the `- 10` for?

Reply:
It's an arbitrary padding between the dropdown menu and the window. 

When the dropdown is opened we calculate the maximum available space between the Select frame and the window to determine the maximum height the dropdown can be. The drop down will never open and expand larger than the available window space. 

![image](https://user-images.githubusercontent.com/6720991/42327029-628a21e8-8030-11e8-80b2-9ccab1bc2e20.png)


The 10 pixels is the distance between the bottom of the dropdown and the window in this picture.